salt/modules/cmdmod.py

```diff
@@ -1027,6 +1031,12 @@ def run(cmd,
 
       .. versionadded:: Fluorine
 
+    :param bool stdin_raw_newlines : False
+        Normally, newlines present in ``stdin`` as ``\\n`` will be 'unescaped',
+        i.e. replaced with a ``\n``. Set this parameter to ``True`` to leave
+        the newlines as-is. This should be used when you are supplying data
+        using ``stdin`` that should not be modified.
+
```

Comment:
This needs a `.. versionadded:: Fluorine` with one line of separation from the above documentation. Additionally, we need the same documentation in _every_ function which invokes `_run()`.

Reply:
Done

Reply:
Thanks!